lib/screens/add_exercise_screen.dart

```diff
@@ -66,56 +75,115 @@ class _AddExerciseScreenState extends State<AddExerciseScreen> {
-            _keys[_currentStep].currentState?.save();
+    Profile? _user = Provider.of<UserProvider>(context, listen: false).profile;
 
-            if (_currentStep != lastStepIndex) {
-              setState(() {
-                _currentStep += 1;
-              });
-            }
-          }
-        },
-        onStepCancel: () => setState(() {
-          if (_currentStep != 0) {
-            _currentStep -= 1;
-          }
-        }),
-        /*
+    return !_user!.emailVerified
+        ? EmailNotVerified(context)
+        : Scaffold(
+            appBar:
+                EmptyAppBar(AppLocalizations.of(context).contributeExercise),
+            body: Stepper(
+              controlsBuilder: _controlsBuilder,
+              steps: [
+                Step(
+                  title: Text(AppLocalizations.of(context).baseData),
+                  content: Step1Basics(formkey: _keys[0]),
+                ),
+                Step(
+                  title: Text(AppLocalizations.of(context).variations),
+                  content: Step2Variations(formkey: _keys[1]),
+                ),
+                Step(
+                  title: Text(AppLocalizations.of(context).description),
+                  content: Step3Description(formkey: _keys[2]),
+                ),
+                Step(
+                  title: Text(AppLocalizations.of(context).translation),
+                  content: Step4Translation(formkey: _keys[3]),
+                ),
+                Step(
+                  title: Text(AppLocalizations.of(context).images),
+                  content: Step5Images(formkey: _keys[4]),
+                ),
+              ],
+              currentStep: _currentStep,
+              onStepContinue: () {
+                if (_keys[_currentStep].currentState?.validate() ?? false) {
+                  _keys[_currentStep].currentState?.save();
+
+                  if (_currentStep != lastStepIndex) {
+                    setState(() {
+                      _currentStep += 1;
+                    });
+                  }
+                }
+              },
+              onStepCancel: () => setState(() {
+                if (_currentStep != 0) {
+                  _currentStep -= 1;
+                }
+              }),
+              /*
         onStepTapped: (int index) {
           setState(() {
             _currentStep = index;
           });
         },
          */
+            ),
+          );
+  }
+
+  Widget EmailNotVerified(context) {
+    Profile? _user = Provider.of<UserProvider>(context, listen: false).profile;
+
+    return Scaffold(
+      appBar: EmptyAppBar('Verify Email'),
+      body: Container(
+        padding: EdgeInsets.all(25),
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Text(
+                'Your profile needs to be verified in order to add any exercise',
```

Comment:
Can you mark the text as translatable? You just need to add an entry to app_en.arb and then you can use it with `AppLocalizations.of(context).foo` (sometimes the IDE doesn't pick this up right away and complains that foo doesn't exists, but the app builds just fine).

I would also suggest that we use the same text as in the web application: "You can only contribute exercises if your account is older than {{days}} days and have verified your email". We can just store the number of days in consts.dart with 14, at the moment this is not configurable.